inst/app/www/reportHtml.Rmd

```diff
@@ -25,6 +25,14 @@ params:
   downloads_plot_data: NA
 ---
 
+<style type="text/css">
+.main-container {
+  max-width: 1200px;
```

Comment:
Might be a bit too nit-picky, but maybe something like `max-width: min(1200px, 80%);` so that margins still exist on small screens? What do you think?